os/compression/compress_read.c

```diff
@@ -281,14 +470,12 @@ int compress_read(int filfd, uint16_t binary_header_size, FAR uint8_t *buffer, s
 	int first_block;
 	int last_block;
 	int no_blocks;
-	int index;
-	int ret;
+	int index;					/* Block number in compressed file */
```

Comment:
Here are two indexs, `index` and `blockcache_index`. Could you rename this with usage like `blockcache_index`?

Reply:
done